src/Templates/TemplateSolutions/CustomTypeEditor/CustomControlLibrary.Design.Server/ControlDesigner/CustomControlDesigner.ActionList.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Microsoft.DotNet.DesignTools.Designers;
+using Microsoft.DotNet.DesignTools.Designers.Actions;
+using System.ComponentModel;
+
+namespace CustomControlLibrary.Designer.Server
+{
+    internal partial class CustomControlDesigner
+    {
+        /// <summary>
+        ///  Action lists implementation for the <see cref="CustomControl"/>.
+        /// </summary>
+        /// <remarks>
+        ///  Note: Action lists for the out-of-process Designer can be implemented exactly as they would be for the in-process
+        ///  Designer, except: The control designer has to be compiled against the Winforms Designer Extensibility SDK, and ActionList
+        ///  related classes must come from the <see cref="Microsoft.DotNet.DesignTools.Designers.Actions"/> namespace.
+        /// </remarks>
+        private class ActionList : DesignerActionList
+        {
+            private const string Behavior = nameof(Behavior);
```

Comment:
This looks inconsistent - we difine a const here but other consts we inline - e.g., `nameof(CustomProperty)`, `nameof(InvokeCustomTypeEditor)`, etc.

Reply:
It's the only name used a) twice and b) where there isn't an existing element you could do a `nameof` on. Leave it like that.